playwright/integration/auth.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from "../support/test-utils";
+import {
+  API_URL,
+  FULFILL_PATH,
+  LIMITED_ACCESS_WORK_PATH,
+  NYPL_LOGIN_URL,
+} from "~/mocks/mockEnv";
+import { server } from "~/mocks/server";
+
+test.beforeEach(async ({ context }) => {
+  await context.clearCookies();
+  server.listen({
+    onUnhandledRequest: "bypass",
+  });
+});
+test.afterEach(() => server.resetHandlers());
+test.afterAll(() => server.close());
+
+test.describe("Cookie authentication", () => {
+  test("redirects to NYPL log in page with no cookie", async ({
+    page,
+    port,
+  }) => {
+    await page.goto(`http://localhost:${port}${LIMITED_ACCESS_WORK_PATH}`);
+    await page.getByRole("link", { name: "title Download PDF" }).click();
+    await page.waitForURL(`**${NYPL_LOGIN_URL}**`);
+    const url = new URL(page.url());
+    const redirectUri = url.searchParams.get("redirect_uri");
+    expect(redirectUri).toContain(LIMITED_ACCESS_WORK_PATH);
```

Comment:
A suggestion here to make this super clear is to add a new line between setup and the expectation/assertion